.github/workflows/test-docker.yml

```diff
@@ -50,7 +50,6 @@ jobs:
           fedora-39-amd64,
           fedora-40-amd64,
           gentoo,
-          ubuntu-20.04-focal-amd64,
```

Comment:
We could either update the image to upgrade the default Python version, but given Pillow 11.0 is set for release in October 2024, it's not that long until Ubuntu 20.04's EOL in April 2025 so we could just remove it?

I don't mind too much either way.